lib/notifications/display.dart

```diff
@@ -25,6 +25,10 @@ AndroidNotificationHostApi get _androidHost => ZulipBinding.instance.androidNoti
 
 /// Service for configuring our Android "notification channel".
 class NotificationChannelManager {
+  /// The channel ID we use for our one notification channel, which we use for
+  /// all notifications.
+  // TODO(?) at launch make sure this channel-id doesn't collide with the one
+  // present in zulip-mobile.
```

Comment:
More accurately, I think, the collision to avoid is with a channel ID that's active in the installed app the user will be upgrading from. In theory, that could be different from the one zulip-mobile has in `main`, right, because the user could be upgrading from a very old zulip-mobile straight to zulip-flutter?

I see in `main` zulip-mobile has "messages-3". There's a handy list of previous values in a code comment on it:

```kotlin
/** The channel ID we use for our one notification channel, which we use for all notifications. */
// Previous values: "default", "messages-1", (alpha-only: "messages-2")
val CHANNEL_ID = "messages-3"
```

which…hmm, that list does include "messages-1". I'm not sure how old that one is, and how many of our users will still have that as their active channel.

Rather than tracking that down, maybe simplest to be conservative and just say something like

```dart
  // TODO(launch) check this doesn't match zulip-mobile's current or previous channel IDs
```